workspaces/global-floating-action-button/plugins/global-floating-action-button/src/plugin.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  createComponentExtension,
+  createPlugin,
+  createRoutableExtension,
+} from '@backstage/core-plugin-api';
+import { createRouteRef } from '@backstage/core-plugin-api';
+
+const testRouteRef = createRouteRef({
+  id: 'test-global-floating-action-button',
+});
```

Comment:
Similar to header, I guess we can live without any route

Reply:
I created this Route for the test environment, the floating button plugin has no route